Error installing relays

When running `solo relay install` it throws error saying: `prompts.promptSelectNamespaceArg is not a function`

```
******************************* Solo *********************************************
Version                 : 0.22.1
Kubernetes Context      : kind-solo-e2e
Kubernetes Cluster      : kind-solo-e2e
Kubernetes Namespace    : solo
**********************************************************************************
(node:41459) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
✖ prompts.promptSelectNamespaceArg is not a function
◼ Prepare chart values
◼ Install JSON RPC Relay
*********************************** ERROR *****************************************
Error installing relays
***********************************************************************************

```

relays is failing to run:
``` 
> start                                                                                                              │
│ > npx lerna exec --scope @hashgraph/json-rpc-server -- npm run start                                                 │
│                                                                                                                      │
│ lerna notice cli v8.0.1                                                                                              │
│ lerna info versioning independent                                                                                    │
│ lerna notice filter including "@hashgraph/json-rpc-server"                                                           │
│ lerna info filter [ '@hashgraph/json-rpc-server' ]                                                                   │
│ lerna info Executing command in 1 package: "npm run start"                                                           │
│                                                                                                                      │
│ > @hashgraph/json-rpc-server@0.39.1 start                                                                            │
│ > node dist/index.js                                                                                                 │
│                                                                                                                      │
│ Killed                                                                                                               │
│ npm ERR! Lifecycle script `start` failed with error:                                                                 │
│ npm ERR! Error: command failed                                                                                       │
│ npm ERR!   in workspace: @hashgraph/json-rpc-server@0.39.1                                                           │
│ npm ERR!   at location: /home/node/app/packages/server                                                               │
│ lerna ERR! npm run start exited 1 in '@hashgraph/json-rpc-server'                                                    │
│ lerna ERR! npm run start exited 1 in '@hashgraph/json-rpc-server'                                                    │
│ npm notice                                                                                                           │
│ npm notice New major version of npm available! 9.8.1 -> 10.5.0                                                       │
│ npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.5.0>                                              │
│ npm notice Run `npm install -g npm@10.5.0` to update!                                                                │
│ npm notice                                                                                                           │
│ Stream closed EOF for solo/relay-node0-hedera-json-rpc-relay-658665dbd4-9hxnl (hedera-json-rpc-relay)                │
```

--- a/DEV.md
+++ b/DEV.md
@@ -25,8 +25,8 @@ Below we describe how you can set up local environment and contribute to `solo`.
   * Run `npm run test-e2e`
 
 * Tests are run in random order. The random seed value is shown as message such as:
-`Using timestamp seed 1711414247085 for random test order`
+  `Using timestamp seed 1711414247085 for random test order`
 
 * If you like to rerun tests with the same seed, use environment variable `RANDOM_SEED=<integer_number>` with `npm run test-e2e` command.
-  * Example: `RANDOM_SEED=20 npm run test-e2e`, 
-  and you should see an output like: `Using preset seed 20 for random test order`
+  * Example: `RANDOM_SEED=20 npm run test-e2e`,
+    and you should see an output like: `Using preset seed 20 for random test order`

--- a/README.md
+++ b/README.md
@@ -124,7 +124,7 @@ You may now view pods in your cluster using `k9s -A` as below:
 $ solo init -t v0.42.5 -i node0,node1,node2 -n "${SOLO_NAMESPACE}" -s "${SOLO_CLUSTER_SETUP_NAMESPACE}" --key-format pfx 
 
 ******************************* Solo *********************************************
-Version                 : 0.22.0
+Version                 : 0.22.1
 Kubernetes Context      : kind-solo
 Kubernetes Cluster      : kind-solo
 Kubernetes Namespace    : solo
@@ -152,7 +152,7 @@ We need to generate `pfx` keys as `pem` key files are only supported by Hedera p
 $ solo node keys --gossip-keys --tls-keys --key-format pfx 
 
 ******************************* Solo *********************************************
-Version                 : 0.22.0
+Version                 : 0.22.1
 Kubernetes Context      : kind-solo
 Kubernetes Cluster      : kind-solo
 Kubernetes Namespace    : solo
@@ -175,7 +175,7 @@ hedera-node0.key  hedera-node1.key  hedera-node2.key  private-node1.pfx public.p
 $ solo cluster setup
 
 ******************************* Solo *********************************************
-Version                 : 0.22.0
+Version                 : 0.22.1
 Kubernetes Context      : kind-solo
 Kubernetes Cluster      : kind-solo
 Kubernetes Namespace    : solo
@@ -194,7 +194,7 @@ Kubernetes Namespace    : solo
 $ solo network deploy
 
 ******************************* Solo *********************************************
-Version                 : 0.22.0
+Version                 : 0.22.1
 Kubernetes Context      : kind-solo
 Kubernetes Cluster      : kind-solo
 Kubernetes Namespace    : solo
@@ -215,7 +215,7 @@ Kubernetes Namespace    : solo
 $ solo node setup
 
 ******************************* Solo *********************************************
-Version                 : 0.22.0
+Version                 : 0.22.1
 Kubernetes Context      : kind-solo
 Kubernetes Cluster      : kind-solo
 Kubernetes Namespace    : solo
@@ -262,12 +262,12 @@ Kubernetes Namespace    : solo
 $ solo node start
 
 ******************************* Solo *********************************************
-Version                 : 0.22.0
+Version                 : 0.22.1
 Kubernetes Context      : kind-solo
 Kubernetes Cluster      : kind-solo
 Kubernetes Namespace    : solo
 **********************************************************************************
-✔ Initialize [0.8s]
+✔ Initialize [0.1s]
 ✔ Identify network pods
   ✔ Check network pod: node0
   ✔ Check network pod: node1
@@ -280,93 +280,74 @@ Kubernetes Namespace    : solo
   ✔ Check node: node0 [24s]
   ✔ Check node: node1 [0.1s]
   ✔ Check node: node2 [0.1s]
-✔ Enable mirror node [33s]
-  ✔ Check node proxies are ACTIVE [7s]
-    ✔ Check proxy for node: node0 [2s]
-    ✔ Check proxy for node: node1 [1s]
-    ✔ Check proxy for node: node2 [4s]
-  ✔ Prepare address book [0.2s]
-  ✔ Deploy mirror node [3s]
-  ✔ Waiting for Hedera Explorer to be ready [22s]
-✔ Update special account keys [54s]
-  ✔ Prepare for account key updates
-  ✔ Update special account key sets [54s]
-    ✔ Updating set 1 of 30 [0.7s]
-    ✔ Updating set 2 of 30 [2s]
-    ✔ Updating set 3 of 30 [1s]
-    ✔ Updating set 4 of 30 [2s]
-    ✔ Updating set 5 of 30 [1s]
-    ✔ Updating set 6 of 30 [1s]
-    ✔ Updating set 7 of 30 [2s]
-    ✔ Updating set 8 of 30 [1s]
-    ✔ Updating set 9 of 30 [1s]
-    ✔ Updating set 10 of 30 [1s]
-    ✔ Updating set 11 of 30 [1s]
-    ✔ Updating set 12 of 30 [1s]
-    ✔ Updating set 13 of 30 [1s]
-    ✔ Updating set 14 of 30 [2s]
-    ✔ Updating set 15 of 30 [2s]
-    ✔ Updating set 16 of 30 [1s]
-    ✔ Updating set 17 of 30 [1s]
-    ✔ Updating set 18 of 30 [1s]
-    ✔ Updating set 19 of 30 [4s]
-    ✔ Updating set 20 of 30 [1s]
-    ✔ Updating set 21 of 30 [1s]
-    ✔ Updating set 22 of 30 [1s]
-    ✔ Updating set 23 of 30 [1s]
-    ✔ Updating set 24 of 30 [1s]
-    ✔ Updating set 25 of 30 [1s]
-    ✔ Updating set 26 of 30 [1s]
-    ✔ Updating set 27 of 30 [2s]
-    ✔ Updating set 28 of 30 [1s]
-    ✔ Updating set 29 of 30 [1s]
-    ✔ Updating set 30 of 30 [0.5s]
-  ✔ Display results
+✔ Check node proxies are ACTIVE [0.1s]
+  ✔ Check proxy for node: node0
+  ✔ Check proxy for node: node1
+  ✔ Check proxy for node: node2
+```
 
-> upgrading chart: fullstack-deployment
-OK chart 'fullstack-deployment' is upgraded
-Account keys updated SUCCESSFULLY: 701
+* Deploy mirror node
+
+```
+$ solo mirror-node deploy
+******************************* Solo *********************************************
+Version                 : 0.22.1
+Kubernetes Context      : kind-solo-e2e
+Kubernetes Cluster      : kind-solo-e2e
+Kubernetes Namespace    : solo
+**********************************************************************************
+✔ Initialize
+✔ Enable mirror-node [4s]
+  ✔ Prepare address book [0.1s]
+  ✔ Deploy mirror-node [3s]
+✔ Check Mirror node components are ACTIVE [3s]
+  ✔ Check Postgres DB [3s]
+  ✔ Check Importer
+  ✔ Check REST API
+  ✔ Check Web3
+  ✔ Check GRPC
+  ✔ Check Hedera Explorer
 ```
 
 You may view the list of pods using `k9s` as below:
 
 ```
- Context: kind-solo                                <0> all       <a>      Attac… ____  __.________
- Cluster: kind-solo                                <1> default   <ctrl-d> Delete|    |/ _/   __   \______
- User:    kind-solo                                              <d>      Descri|      < \____    /  ___/
- K9s Rev: v0.27.4 ⚡️v0.32.3                                      <e>      Edit  |    |  \   /    /\___ \
- K8s Rev: v1.27.3                                                <?>      Help  |____|__ \ /____//____  >
- CPU:     n/a                                                    <ctrl-k> Kill          \/            \/
+ Context: kind-solo-e2e                            <0> all       <a>      Attach     <l>     … ____  __.________
+ Cluster: kind-solo-e2e                            <1> default   <ctrl-d> Delete     <p>      |    |/ _/   __   \______
+ User:    kind-solo-e2e                                          <d>      Describe   <shift-f>|      < \____    /  ___/
+ K9s Rev: v0.27.4 ⚡️v0.32.4                                      <e>      Edit       <s>      |    |  \   /    /\___ \
+ K8s Rev: v1.27.3                                                <?>      Help       <n>      |____|__ \ /____//____  >
+ CPU:     n/a                                                    <ctrl-k> Kill       <f>              \/            \/
  MEM:     n/a
-┌──────────────────────────────────────────── Pods(all)[27] ─────────────────────────────────────────────┐
-│ NAMESPACE↑          NAME                                                   PF READY RESTARTS STATUS    │
-│ kube-system         coredns-5d78c9869d-kc27p                               ●  1/1          0 Running   │
-│ kube-system         coredns-5d78c9869d-r8mzz                               ●  1/1          0 Running   │
-│ kube-system         etcd-solo-control-plane                                ●  1/1          0 Running   │
-│ kube-system         kindnet-gppbk                                          ●  1/1          0 Running   │
-│ kube-system         kube-apiserver-solo-control-plane                      ●  1/1          0 Running   │
-│ kube-system         kube-controller-manager-solo-control-plane             ●  1/1          0 Running   │
-│ kube-system         kube-proxy-wb9w5                                       ●  1/1          0 Running   │
-│ kube-system         kube-scheduler-solo-control-plane                      ●  1/1          0 Running   │
-│ local-path-storage  local-path-provisioner-6bc4bddd6b-5vh5d                ●  1/1          0 Running   │
-│ solo                envoy-proxy-node0-84947f844f-5j4fc                     ●  1/1          0 Running   │
-│ solo                envoy-proxy-node1-65f8879dcc-ztmnw                     ●  1/1          0 Running   │
-│ solo                envoy-proxy-node2-667f848689-bj849                     ●  1/1          0 Running   │
-│ solo                fullstack-deployment-grpc-69f9cc5666-744mm             ●  1/1          0 Running   │
-│ solo                fullstack-deployment-hedera-explorer-79f79b7df4-w2bl4  ●  1/1          0 Running   │
-│ solo                fullstack-deployment-importer-65bb89757f-vqg2l         ●  1/1          0 Running   │
-│ solo                fullstack-deployment-postgres-postgresql-0             ●  1/1          0 Running   │
-│ solo                fullstack-deployment-rest-584f5cb6bb-v7gvj             ●  1/1          0 Running   │
-│ solo                fullstack-deployment-web3-69dcdfc4fb-rxzsc             ●  1/1          0 Running   │
-│ solo                haproxy-node0-96f8df6d-dcv88                           ●  1/1          0 Running   │
-│ solo                haproxy-node1-845fb68f48-z492b                         ●  1/1          0 Running   │
-│ solo                haproxy-node2-867656ff6-npgjc                          ●  1/1          0 Running   │
-│ solo                minio-pool-1-0                                         ●  2/2          1 Running   │
-│ solo                network-node0-0                                        ●  5/5          0 Running   │
-│ solo                network-node1-0                                        ●  5/5          0 Running   │
-│ solo                network-node2-0                                        ●  5/5          0 Running   │
-│ solo-cluster        console-557956d575-pdkqw                               ●  1/1          0 Running   │
-│ solo-cluster        minio-operator-7d575c5f84-8pjmf                        ●  1/1          0 Running   │
+┌─────────────────────────────────────────────────── Pods(all)[27] ────────────────────────────────────────────────────┐
+│ NAMESPACE↑          NAME                                                   PF READY RESTARTS STATUS   IP             │
+│ fullstack-setup     console-557956d575-fqctd                               ●  1/1          0 Running  10.244.0.4     │
+│ fullstack-setup     minio-operator-7d575c5f84-j9p6f                        ●  1/1          0 Running  10.244.0.3     │
+│ kube-system         coredns-5d78c9869d-gknqp                               ●  1/1          0 Running  10.244.0.6     │
+│ kube-system         coredns-5d78c9869d-q59pc                               ●  1/1          0 Running  10.244.0.5     │
+│ kube-system         etcd-solo-e2e-control-plane                            ●  1/1          0 Running  172.18.0.2     │
+│ kube-system         kindnet-w9ps5                                          ●  1/1          0 Running  172.18.0.2     │
+│ kube-system         kube-apiserver-solo-e2e-control-plane                  ●  1/1          0 Running  172.18.0.2     │
+│ kube-system         kube-controller-manager-solo-e2e-control-plane         ●  1/1          0 Running  172.18.0.2     │
+│ kube-system         kube-proxy-p69z8                                       ●  1/1          0 Running  172.18.0.2     │
+│ kube-system         kube-scheduler-solo-e2e-control-plane                  ●  1/1          0 Running  172.18.0.2     │
+│ local-path-storage  local-path-provisioner-6bc4bddd6b-8pkfk                ●  1/1          0 Running  10.244.0.2     │
+│ solo                envoy-proxy-node0-84947f844f-f28tp                     ●  1/1          0 Running  10.244.0.215   │
+│ solo                envoy-proxy-node1-65f8879dcc-j2lrk                     ●  1/1          0 Running  10.244.0.216   │
+│ solo                envoy-proxy-node2-667f848689-dkmf9                     ●  1/1          0 Running  10.244.0.214   │
+│ solo                fullstack-deployment-grpc-69f9cc5666-lf6ql             ●  1/1          0 Running  10.244.0.227   │
+│ solo                fullstack-deployment-hedera-explorer-79f79b7df4-wjdct  ●  1/1          0 Running  10.244.0.226   │
+│ solo                fullstack-deployment-importer-864489ffb8-6v8tk         ●  1/1          0 Running  10.244.0.228   │
+│ solo                fullstack-deployment-postgres-postgresql-0             ●  1/1          0 Running  10.244.0.232   │
+│ solo                fullstack-deployment-rest-584f5cb6bb-q9vnt             ●  1/1          0 Running  10.244.0.230   │
+│ solo                fullstack-deployment-web3-69dcdfc4fb-mm5pk             ●  1/1          0 Running  10.244.0.229   │
+│ solo                haproxy-node0-6969f76c77-n5cfl                         ●  1/1          1 Running  10.244.0.219   │
+│ solo                haproxy-node1-59f6976d45-x6xmp                         ●  1/1          1 Running  10.244.0.217   │
+│ solo                haproxy-node2-6df64d5457-hf9ps                         ●  1/1          1 Running  10.244.0.218   │
+│ solo                minio-pool-1-0                                         ●  2/2          1 Running  10.244.0.224   │
+│ solo                network-node0-0                                        ●  5/5          0 Running  10.244.0.221   │
+│ solo                network-node1-0                                        ●  5/5          0 Running  10.244.0.222   │
+│ solo                network-node2-0                                        ●  5/5          0 Running  10.244.0.220   │
 ```
 
 #### Access Hedera Network services
@@ -377,43 +358,21 @@ Once the nodes are up, you may now expose various services (using `k9s` (shift-f
 * HAProxy: `haproxy-<node ID>-svc`
 * Envoy Proxy: `envoy-proxy-<node ID>-svc`
 * Hedera explorer: `fullstack-deployment-hedera-explorer`
+* JSON Rpc Relays
+  * You can deploy JSON RPC relays for one or more nodes as below:
+  ```
+  $ solo relay deploy -i node0,node1 
 
-```
-Context: kind-solo                                <0> all       <ctrl-l> Bench Run/Stop   <p>       Logs Previous                                                                           ____  __.________
- Cluster: kind-solo                                <1> default   <ctrl-d> Delete           <shift-f> Port-Forward                                                                           |    |/ _/   __   \______
- User:    kind-solo                                              <d>      Describe         <y>       YAML                                                                                   |      < \____    /  ___/
- K9s Rev: v0.27.4 ⚡️v0.32.3                                      <e>      Edit                                                                                                              |    |  \   /    /\___ \
- K8s Rev: v1.27.3                                                <?>      Help                                                                                                              |____|__ \ /____//____  >
- CPU:     n/a                                                    <l>      Logs                                                                                                                      \/            \/
- MEM:     n/a
-┌──────────────────────────────────────────────────────────────────────────────────────────────── Services(all)[24] ─────────────────────────────────────────────────────────────────────────────────────────────────┐
-│ NAMESPACE↑    NAME                                               TYPE           CLUSTER-IP      EXTERNAL-IP  PORTS                                                                                        AGE      │
-│ default       kubernetes                                         ClusterIP      10.96.0.1                    https:443►0                                                                                  58m      │
-│ kube-system   kube-dns                                           ClusterIP      10.96.0.10                   dns:53►0╱UDP dns-tcp:53►0 metrics:9153►0                                                     58m      │
-│ solo          envoy-proxy-node0-svc                              ClusterIP      10.96.108.35                 hedera-grpc-web:8080►0 prometheus:9090►0                                                     5m43s    │
-│ solo          envoy-proxy-node1-svc                              ClusterIP      10.96.28.255                 hedera-grpc-web:8080►0 prometheus:9090►0                                                     5m43s    │
-│ solo          envoy-proxy-node2-svc                              ClusterIP      10.96.114.202                hedera-grpc-web:8080►0 prometheus:9090►0                                                     5m43s    │
-│ solo          fullstack-deployment-grpc                          ClusterIP      10.96.189.209                grpc:5600►0 http:80►0                                                                        3m37s    │
-│ solo          fullstack-deployment-hedera-explorer               ClusterIP      10.96.169.9                  http:80►0                                                                                    3m37s    │
-│ solo          fullstack-deployment-postgres-pgpool               ClusterIP      10.96.189.235                postgresql:5432►0                                                                            3m37s    │
-│ solo          fullstack-deployment-postgres-postgresql           ClusterIP      10.96.92.24                  postgresql:5432►0                                                                            3m37s    │
-│ solo          fullstack-deployment-postgres-postgresql-headless  ClusterIP                                   postgresql:5432►0                                                                            3m37s    │
-│ solo          fullstack-deployment-rest                          ClusterIP      10.96.247.93                 http:80►0                                                                                    3m37s    │
-│ solo          fullstack-deployment-web3                          ClusterIP      10.96.254.144                http:80►0                                                                                    3m37s    │
-│ solo          haproxy-node0-svc                                  LoadBalancer   10.96.245.12    <pending>    non-tls-grpc-client-port:50211►31715 tls-grpc-client-port:50212►31619 prometheus:9090►30237  5m43s    │
-│ solo          haproxy-node1-svc                                  LoadBalancer   10.96.132.20    <pending>    non-tls-grpc-client-port:50211►30805 tls-grpc-client-port:50212►30238 prometheus:9090►30936  5m43s    │
-│ solo          haproxy-node2-svc                                  LoadBalancer   10.96.171.41    <pending>    non-tls-grpc-client-port:50211►32476 tls-grpc-client-port:50212►31387 prometheus:9090►30776  5m43s    │
-│ solo          minio                                              ClusterIP      10.96.65.174                 http-minio:80►0                                                                              5m38s    │
-│ solo          minio-console                                      ClusterIP      10.96.211.100                http-console:9090►0                                                                          5m38s    │
-│ solo          minio-hl                                           ClusterIP                                   http-minio:9000►0                                                                            5m38s    │
-│ solo          network-node0-svc                                  ClusterIP      10.96.141.1                  gossip:50111►0 grpc-non-tls:50211►0 grpc-tls:50212►0 prometheus:9090►0                       5m43s    │
-│ solo          network-node1-svc                                  ClusterIP      10.96.92.144                 gossip:50111►0 grpc-non-tls:50211►0 grpc-tls:50212►0 prometheus:9090►0                       5m43s    │
-│ solo          network-node2-svc                                  ClusterIP      10.96.140.243                gossip:50111►0 grpc-non-tls:50211►0 grpc-tls:50212►0 prometheus:9090►0                       5m43s    │
-│ solo-cluster  console                                            ClusterIP      10.96.31.144                 http:9090►0 https:9443►0                                                                     54m      │
-│ solo-cluster  operator                                           ClusterIP      10.96.212.116                http:4221►0                                                                                  54m      │
-│ solo-cluster  sts                                                ClusterIP      10.96.108.130                https:4223►0                                                                                 54m      │
-│ 
-```
+  ******************************* Solo *********************************************
+  Version                 : 0.22.1
+  Kubernetes Context      : kind-solo-e2e
+  Kubernetes Cluster      : kind-solo-e2e
+  Kubernetes Namespace    : solo
+  **********************************************************************************
+  ✔ Initialize
+  ✔ Prepare chart values
+  ✔ Deploy JSON RPC Relay [1s]
+  ```
 
 ### Example - 2: Deploy a standalone test network (version `0.47.0-alpha.0`)
 

--- a/src/commands/base.mjs
+++ b/src/commands/base.mjs
@@ -19,17 +19,17 @@ import { MissingArgumentError } from '../core/errors.mjs'
 import { ShellRunner } from '../core/shell_runner.mjs'
 
 export class BaseCommand extends ShellRunner {
-  async prepareChartPath (chartDir, chartRepo, chartName) {
+  async prepareChartPath (chartDir, chartRepo, chartReleaseName) {
     if (!chartRepo) throw new MissingArgumentError('chart repo name is required')
-    if (!chartName) throw new MissingArgumentError('chart name is required')
+    if (!chartReleaseName) throw new MissingArgumentError('chart name is required')
 
     if (chartDir) {
-      const chartPath = `${chartDir}/${chartName}`
+      const chartPath = `${chartDir}/${chartReleaseName}`
       await this.helm.dependency('update', chartPath)
       return chartPath
     }
 
-    return `${chartRepo}/${chartName}`
+    return `${chartRepo}/${chartReleaseName}`
   }
 
   constructor (opts) {

--- a/src/commands/base.mjs
+++ b/src/commands/base.mjs
@@ -21,7 +21,7 @@ import { ShellRunner } from '../core/shell_runner.mjs'
 export class BaseCommand extends ShellRunner {
   async prepareChartPath (chartDir, chartRepo, chartReleaseName) {
     if (!chartRepo) throw new MissingArgumentError('chart repo name is required')
-    if (!chartReleaseName) throw new MissingArgumentError('chart name is required')
+    if (!chartReleaseName) throw new MissingArgumentError('chart release name is required')
 
     if (chartDir) {
       const chartPath = `${chartDir}/${chartReleaseName}`

--- a/src/commands/flags.mjs
+++ b/src/commands/flags.mjs
@@ -170,8 +170,8 @@ export const releaseTag = {
 export const relayReleaseTag = {
   name: 'relay-release',
   definition: {
-    describe: 'Relay release tag to be used (e.g. v0.44.0)',
-    defaultValue: '',
+    describe: 'Relay release tag to be used (e.g. v0.43.0)',
+    defaultValue: 'v0.43.0',
     type: 'string'
   }
 }

--- a/src/commands/flags.mjs
+++ b/src/commands/flags.mjs
@@ -170,7 +170,7 @@ export const releaseTag = {
 export const relayReleaseTag = {
   name: 'relay-release',
   definition: {
-    describe: 'Relay release tag to be used (e.g. v0.39.1)',
+    describe: 'Relay release tag to be used (e.g. v0.44.0)',
     defaultValue: '',
     type: 'string'
   }

--- a/src/commands/relay.mjs
+++ b/src/commands/relay.mjs
@@ -15,11 +15,12 @@
  *
  */
 import { Listr } from 'listr2'
+import * as paths from 'path'
 import { FullstackTestingError, MissingArgumentError } from '../core/errors.mjs'
+import * as helpers from '../core/helpers.mjs'
+import { constants } from '../core/index.mjs'
 import { BaseCommand } from './base.mjs'
 import * as flags from './flags.mjs'
-import * as paths from 'path'
-import { constants } from '../core/index.mjs'
 import * as prompts from './prompts.mjs'
 
 export class RelayCommand extends BaseCommand {
@@ -80,53 +81,53 @@ export class RelayCommand extends BaseCommand {
     return releaseName
   }
 
-  async install (argv) {
+  async deploy (argv) {
     const self = this
     const tasks = new Listr([
       {
         title: 'Initialize',
         task: async (ctx, task) => {
+          // reset nodeID
+          self.configManager.setFlag(flags.nodeIDs, '')
+
           self.configManager.update(argv)
 
-          // extract config values
-          const valuesFile = self.configManager.getFlag(flags.valuesFile)
-          const nodeIds = self.configManager.getFlag(flags.nodeIDs)
-          const chainId = self.configManager.getFlag(flags.chainId)
-          const relayRelease = self.configManager.getFlag(flags.relayReleaseTag)
-          const replicaCount = self.configManager.getFlag(flags.replicaCount)
-          const operatorId = self.configManager.getFlag(flags.operatorId)
-          const operatorKey = self.configManager.getFlag(flags.operatorKey)
-
-          const namespace = self.configManager.getFlag(flags.namespace)
-          const chartDir = self.configManager.getFlag(flags.chartDirectory)
+          await prompts.execute(task, self.configManager, [
+            flags.chartDirectory,
+            flags.namespace,
+            flags.valuesFile,
+            flags.nodeIDs,
+            flags.chainId,
+            flags.relayReleaseTag,
+            flags.replicaCount,
+            flags.operatorId,
+            flags.operatorKey
+          ])
 
           // prompt if inputs are empty and set it in the context
-          const namespaces = await self.k8.getNamespaces()
           ctx.config = {
-            chartDir: await prompts.promptChartDir(task, chartDir),
-            namespace: await prompts.promptSelectNamespaceArg(task, namespace, namespaces),
-            valuesFile: await prompts.promptValuesFile(task, valuesFile),
-            nodeIds: await prompts.promptNodeIds(task, nodeIds),
-            chainId: await prompts.promptChainId(task, chainId),
-            relayRelease: await prompts.promptRelayReleaseTag(task, relayRelease),
-            replicaCount: await prompts.promptReplicaCount(task, replicaCount),
-            operatorId: await prompts.promptOperatorId(task, operatorId),
-            operatorKey: await prompts.promptOperatorId(task, operatorKey)
+            chartDir: self.configManager.getFlag(flags.chartDirectory),
+            namespace: self.configManager.getFlag(flags.namespace),
+            valuesFile: self.configManager.getFlag(flags.valuesFile),
+            nodeIds: helpers.parseNodeIDs(self.configManager.getFlag(flags.nodeIDs)),
+            chainId: self.configManager.getFlag(flags.chainId),
+            relayRelease: self.configManager.getFlag(flags.relayReleaseTag),
+            replicaCount: self.configManager.getFlag(flags.replicaCount),
+            operatorId: self.configManager.getFlag(flags.operatorId),
+            operatorKey: self.configManager.getFlag(flags.operatorKey)
           }
 
-          self.logger.debug('Finished prompts', { ctx })
+          ctx.releaseName = self.prepareReleaseName(ctx.config.nodeIds)
+          ctx.isChartInstalled = await self.chartManager.isChartInstalled(ctx.config.namespace, ctx.releaseName)
 
-          ctx.releaseName = this.prepareReleaseName(ctx.config.nodeIds)
-          ctx.isChartInstalled = await this.chartManager.isChartInstalled(ctx.config.namespace, ctx.releaseName)
-
-          self.logger.debug('Finished ctx initialization', { ctx })
+          self.logger.debug('Initialized config', { config: ctx.config })
         }
       },
       {
         title: 'Prepare chart values',
         task: async (ctx, _) => {
-          ctx.chartPath = await this.prepareChartPath(ctx.config.chartDir, constants.JSON_RPC_RELAY_CHART, constants.CHART_JSON_RPC_RELAY_NAME)
-          ctx.valuesArg = this.prepareValuesArg(
+          ctx.chartPath = await self.prepareChartPath(ctx.config.chartDir, constants.JSON_RPC_RELAY_CHART, constants.JSON_RPC_RELAY_CHART)
+          ctx.valuesArg = self.prepareValuesArg(
             ctx.config.valuesFile,
             ctx.config.nodeIds,
             ctx.config.chainId,
@@ -138,21 +139,25 @@ export class RelayCommand extends BaseCommand {
         }
       },
       {
-        title: 'Install JSON RPC Relay',
+        title: 'Deploy JSON RPC Relay',
         task: async (ctx, _) => {
           const namespace = ctx.config.namespace
           const releaseName = ctx.releaseName
           const chartPath = ctx.chartPath
           const valuesArg = ctx.valuesArg
 
-          await this.chartManager.install(namespace, releaseName, chartPath, '', valuesArg)
+          await self.chartManager.install(namespace, releaseName, chartPath, '', valuesArg)
 
-          await this.k8.waitForPod(constants.POD_STATUS_RUNNING, [
+          await self.k8.waitForPod(constants.POD_STATUS_RUNNING, [
             'app=hedera-json-rpc-relay',
             `app.kubernetes.io/instance=${releaseName}`
           ], 1, 120, 1000)
 
           this.logger.showList('Deployed Relays', await self.chartManager.getInstalledCharts(namespace))
+
+          // reset nodeID
+          self.configManager.setFlag(flags.nodeIDs, '')
+          self.configManager.persist()
         }
       }
     ], {
@@ -169,40 +174,51 @@ export class RelayCommand extends BaseCommand {
     return true
   }
 
-  async uninstall (argv) {
+  async destroy (argv) {
     const self = this
 
     const tasks = new Listr([
       {
         title: 'Initialize',
         task: async (ctx, task) => {
-          self.configManager.update(argv)
+          // reset nodeID
+          self.configManager.setFlag(flags.nodeIDs, '')
 
-          // extract config values
-          const nodeIds = self.configManager.getFlag(flags.nodeIDs)
-          const namespace = self.configManager.getFlag(flags.namespace)
+          self.configManager.update(argv)
+          await prompts.execute(task, self.configManager, [
+            flags.chartDirectory,
+            flags.namespace,
+            flags.nodeIDs
+          ])
 
           // prompt if inputs are empty and set it in the context
-          const namespaces = await self.k8.getNamespaces()
           ctx.config = {
-            namespace: await prompts.promptSelectNamespaceArg(task, namespace, namespaces),
-            nodeIds: await prompts.promptNodeIds(task, nodeIds)
+            chartDir: self.configManager.getFlag(flags.chartDirectory),
+            namespace: self.configManager.getFlag(flags.namespace),
+            nodeIds: helpers.parseNodeIDs(self.configManager.getFlag(flags.nodeIDs))
           }
 
-          ctx.config.releaseName = this.prepareReleaseName(ctx.config.nodeIds)
-          self.logger.debug('Finished ctx initialization', { ctx })
+          ctx.releaseName = this.prepareReleaseName(ctx.config.nodeIds)
+          ctx.isChartInstalled = await this.chartManager.isChartInstalled(ctx.config.namespace, ctx.releaseName)
+
+          self.logger.debug('Initialized config', { config: ctx.config })
         }
       },
       {
-        title: 'Install JSON RPC Relay',
+        title: 'Destroy JSON RPC Relay',
         task: async (ctx, _) => {
           const namespace = ctx.config.namespace
-          const releaseName = ctx.config.releaseName
+          const releaseName = ctx.releaseName
 
           await this.chartManager.uninstall(namespace, releaseName)
 
-          this.logger.showList('Deployed Relays', await self.chartManager.getInstalledCharts(namespace))
-        }
+          this.logger.showList('Destroyed Relays', await self.chartManager.getInstalledCharts(namespace))
+
+          // reset nodeID
+          self.configManager.setFlag(flags.nodeIDs, '')
+          self.configManager.persist()
+        },
+        skip: (ctx, _) => !ctx.isChartInstalled
       }
     ], {
       concurrent: false,
@@ -225,8 +241,8 @@ export class RelayCommand extends BaseCommand {
       builder: yargs => {
         return yargs
           .command({
-            command: 'install',
-            desc: 'Install a JSON RPC relay',
+            command: 'deploy',
+            desc: 'Deploy a JSON RPC relay',
             builder: y => {
               flags.setCommandFlags(y,
                 flags.namespace,
@@ -243,7 +259,7 @@ export class RelayCommand extends BaseCommand {
             handler: argv => {
               relayCmd.logger.debug("==== Running 'relay install' ===", { argv })
 
-              relayCmd.install(argv).then(r => {
+              relayCmd.deploy(argv).then(r => {
                 relayCmd.logger.debug('==== Finished running `relay install`====')
 
                 if (!r) process.exit(1)
@@ -254,8 +270,8 @@ export class RelayCommand extends BaseCommand {
             }
           })
           .command({
-            command: 'uninstall',
-            desc: 'Uninstall JSON RPC relay',
+            command: 'destroy',
+            desc: 'Destroy JSON RPC relay',
             builder: y => flags.setCommandFlags(y,
               flags.namespace,
               flags.nodeIDs
@@ -264,7 +280,7 @@ export class RelayCommand extends BaseCommand {
               relayCmd.logger.debug("==== Running 'relay uninstall' ===", { argv })
               relayCmd.logger.debug(argv)
 
-              relayCmd.uninstall(argv).then(r => {
+              relayCmd.destroy(argv).then(r => {
                 relayCmd.logger.debug('==== Finished running `relay uninstall`====')
 
                 if (!r) process.exit(1)

--- a/src/core/chart_manager.mjs
+++ b/src/core/chart_manager.mjs
@@ -70,9 +70,9 @@ export class ChartManager {
     return []
   }
 
-  async install (namespaceName, chartName, chartPath, version, valuesArg = '') {
+  async install (namespaceName, chartReleaseName, chartPath, version, valuesArg = '') {
     try {
-      const isInstalled = await this.isChartInstalled(namespaceName, chartName)
+      const isInstalled = await this.isChartInstalled(namespaceName, chartReleaseName)
       if (!isInstalled) {
         let versionArg = ''
         if (version) {
@@ -85,23 +85,23 @@ export class ChartManager {
         }
 
         this.logger.debug(`> installing chart:${chartPath}`)
-        await this.helm.install(`${chartName} ${chartPath} ${versionArg} ${namespaceArg} ${valuesArg}`)
-        this.logger.debug(`OK: chart is installed: ${chartName} (${chartPath})`)
+        await this.helm.install(`${chartReleaseName} ${chartPath} ${versionArg} ${namespaceArg} ${valuesArg}`)
+        this.logger.debug(`OK: chart is installed: ${chartReleaseName} (${chartPath})`)
       } else {
-        this.logger.debug(`OK: chart is already installed:${chartName} (${chartPath})`)
+        this.logger.debug(`OK: chart is already installed:${chartReleaseName} (${chartPath})`)
       }
     } catch (e) {
-      throw new FullstackTestingError(`failed to install chart ${chartName}: ${e.message}`, e)
+      throw new FullstackTestingError(`failed to install chart ${chartReleaseName}: ${e.message}`, e)
     }
 
     return true
   }
 
-  async isChartInstalled (namespaceName, chartName) {
-    this.logger.debug(`> checking if chart is installed [ chart: ${chartName}, namespace: ${namespaceName} ]`)
+  async isChartInstalled (namespaceName, chartReleaseName) {
+    this.logger.debug(`> checking if chart is installed [ chart: ${chartReleaseName}, namespace: ${namespaceName} ]`)
     const charts = await this.getInstalledCharts(namespaceName)
     for (const item of charts) {
-      if (item.startsWith(chartName)) {
+      if (item.startsWith(chartReleaseName)) {
         return true
       }
     }
@@ -109,30 +109,30 @@ export class ChartManager {
     return false
   }
 
-  async uninstall (namespaceName, chartName) {
+  async uninstall (namespaceName, chartReleaseName) {
     try {
-      const isInstalled = await this.isChartInstalled(namespaceName, chartName)
+      const isInstalled = await this.isChartInstalled(namespaceName, chartReleaseName)
       if (isInstalled) {
-        this.logger.debug(`uninstalling chart release: ${chartName}`)
-        await this.helm.uninstall(`-n ${namespaceName} ${chartName}`)
-        this.logger.debug(`OK: chart release is uninstalled: ${chartName}`)
+        this.logger.debug(`uninstalling chart release: ${chartReleaseName}`)
+        await this.helm.uninstall(`-n ${namespaceName} ${chartReleaseName}`)
+        this.logger.debug(`OK: chart release is uninstalled: ${chartReleaseName}`)
       } else {
-        this.logger.debug(`OK: chart release is already uninstalled: ${chartName}`)
+        this.logger.debug(`OK: chart release is already uninstalled: ${chartReleaseName}`)
       }
     } catch (e) {
-      throw new FullstackTestingError(`failed to uninstall chart ${chartName}: ${e.message}`, e)
+      throw new FullstackTestingError(`failed to uninstall chart ${chartReleaseName}: ${e.message}`, e)
     }
 
     return true
   }
 
-  async upgrade (namespaceName, chartName, chartPath, valuesArg = '') {
+  async upgrade (namespaceName, chartReleaseName, chartPath, valuesArg = '') {
     try {
-      this.logger.debug(chalk.cyan('> upgrading chart:'), chalk.yellow(`${chartName}`))
-      await this.helm.upgrade(`-n ${namespaceName} ${chartName} ${chartPath} --reuse-values ${valuesArg}`)
-      this.logger.debug(chalk.green('OK'), `chart '${chartName}' is upgraded`)
+      this.logger.debug(chalk.cyan('> upgrading chart:'), chalk.yellow(`${chartReleaseName}`))
+      await this.helm.upgrade(`-n ${namespaceName} ${chartReleaseName} ${chartPath} --reuse-values ${valuesArg}`)
+      this.logger.debug(chalk.green('OK'), `chart '${chartReleaseName}' is upgraded`)
     } catch (e) {
-      throw new FullstackTestingError(`failed to upgrade chart ${chartName}: ${e.message}`, e)
+      throw new FullstackTestingError(`failed to upgrade chart ${chartReleaseName}: ${e.message}`, e)
     }
 
     return true

